### PR TITLE
Prevented dynamic class loading possibility

### PR DESCRIPTION
**Pull request by mvlasovatl**
It's a fix for the issue allowing to execute arbitrary code from withing the custom notification template renderer.

### CODE REVIEW

File: jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/service/notification/impl/JiraPostFunctionEventRenderer.java [hunk 1]
```diff
@@ -115,6 +115,16 @@ private ChatPostMessageRequestBuilder renderStandardNotification(final JiraPostF
     }
 
     private ChatPostMessageRequestBuilder renderCustomNotification(final JiraPostFunctionEvent event) throws Exception {
+        String customMessageFormat = event.getCustomMessageFormat();
+
+        // prevent attempts to access arbitrary classes by names to execute arbitrary code withing template
+        // Samples of forbidden code:
+        // - "".getClass().forName("com.MyClass")
+        // - "".getClass().getClassLoader().loadClass("com.MyClass")
+        if (customMessageFormat.contains(".forName(") || customMessageFormat.contains(".loadClass(")) {
```
Comment (utluiz) [inline jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/service/notification/impl/JiraPostFunctionEventRenderer.java:124]: I like the simple solution. However, I did some research and found Velocity has a built-in mechanism that would prevent such attacks.

This is an article (weird English, perhaps was translated by some tool?) that talks about the solution: https://titanwolf.org/Network/Articles/Article?AID=e26bf243-336b-4185-ba63-ae974168af9c#gsc.tab=0

This is a random code excerpt that also shows how the configuration can be made: http://svn.contentcontrol.com/ContentControl/tags/release-7.1.0/modules/content-creator/src/main/java/ccc/rendering/velocity/VelocityProcessor.java

TL;DR Add this after line 149:
```
velocityEngine.setProperty("runtime.introspector.uberspect",
            "org.apache.velocity.util.introspection.SecureUberspector");
```

Comment (mvlasovatl) [reply jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/service/notification/impl/JiraPostFunctionEventRenderer.java]: That works! Thanks! I looked for something like this, but missed that article. Standard solution is always better!

**Review by utluiz — APPROVED** [review]
